components/Chat/Chat.tsx

```diff
@@ -152,9 +158,21 @@ export const Chat: FC<Props> = ({
               </>
             ) : (
               <>
-                <div className="flex justify-center border border-b-neutral-300 bg-neutral-100 py-2 text-sm text-neutral-500 dark:border-none dark:bg-[#444654] dark:text-neutral-200">
-                  {t('Model')}: {conversation.model.name}
+              <div className="flex justify-center border border-b-neutral-300 bg-neutral-100 py-2 text-sm text-neutral-500 dark:border-none dark:bg-[#444654] dark:text-neutral-200">
+                {t('Model')}: {conversation.model.name}
+                <IconSettings className="ml-2" onClick={handleSettings} size={21}/>
```

Comment:
`                  <IconSettings
                    className="ml-2 cursor-pointer hover:opacity-50"
                    onClick={handleSettings}
                    size={18}
                  />`
                  
                  If you could just make this change I'll merge :)